packages/core-types/src/SmartAccountTypes.ts

```diff
@@ -140,3 +140,9 @@ export type TransactionBatchDto = {
   transactions: Transaction[]
   chainId?: ChainId
 }
+
+export enum SmartAccountType {
+  CANDIDE,
```

Comment:
Export CANDIDE and SOUL only when we give support of them in sdk.

Reply:
resolved